pkg/products/cloudresources/reconciler.go

```diff
@@ -616,3 +616,13 @@ func (r *Reconciler) reconcileCIDRValue(ctx context.Context, client k8sclient.Cl
 
 	return client.Patch(ctx, cfgMap, k8sclient.Merge)
 }
+
+// createSTSARNSecret create the STS arn secret - should be already validated in preflight checks
+func (r *Reconciler) copySTSSecret(ctx context.Context, client k8sclient.Client, operatorNamespace string) (integreatlyv1alpha1.StatusPhase, error) {
```

Comment:
any benefit to putting this in the STS package?

Reply:
actually wondering if there is any benefit to the function as a whole, if hive is creating the secret in the CRO ns then we don't need this move. Its hanging around since before we came up with the idea to have managed tenants create the secret directly in CRO.

Reply:
will know more after testing

Reply:
actually will refactor to check for the secret instead